test/e2e/json-rpc/wallet_revokePermissions.spec.js

```diff
@@ -0,0 +1,61 @@
+const { strict: assert } = require('assert');
+
+const {
+  withFixtures,
+  defaultGanacheOptions,
+  unlockWallet,
+  openDapp,
+} = require('../helpers');
+const FixtureBuilder = require('../fixture-builder');
+
+describe('Revoke Dapp Permissions', function () {
+  it('should revoke dapp permissions ', async function () {
+    await withFixtures(
+      {
+        dapp: true,
+        fixtures: new FixtureBuilder()
+          .withPermissionControllerConnectedToTestDapp()
+          .build(),
+        ganacheOptions: defaultGanacheOptions,
+        title: this.test?.fullTitle(),
+      },
+      async ({ driver }) => {
+        await driver.navigate();
+        await unlockWallet(driver);
+
+        await openDapp(driver);
+
+        let accountsDiv = await driver.findElement('#accounts');
+
+        assert.equal(
+          await accountsDiv.getText(),
+          '0x5cfe73b6021e818b776b421b1c4db2474086a7e1',
+        );
```

Comment:
This is the bad pattern that can cause a race condition.  Change to:
```suggestion
        let accountsDiv = await driver.findElement({
          text: '0x5cfe73b6021e818b776b421b1c4db2474086a7e1',
          css: '#accounts',
        });
```

Reply:
@HowardBraham  What potential race condition are we trying to avoid by removing an assertion check of the connected address?

Reply:
It may not ever happen in **this** case, but this is the bad pattern: https://consensys.slack.com/archives/GTQAGKY5V/p1686238528522549